packages/auto_mappr/lib/src/extensions/executable_element_extension.dart

```diff
@@ -1,5 +1,12 @@
 import 'package:analyzer/dart/element/element.dart';
 
 extension ExecutableElementExtension on ExecutableElement {
-  String get referCallString => isStatic ? '${enclosingElement.displayName}.$displayName' : displayName;
+          // For top-level functions `isStatic` is true
+          // but the enclosing element has no display name.
+          // This could result in the expression being `.someFunction()` which is invalid.
+          // Therefore, we only add the enclosing element's display name if it is not empty.
```

Comment:
Looks better. Just a last nit: for some reason, there are like 10 leading spaces before comments instead of like 2, and it might be rather a doc comment?

Reply:
Already fixed the formatting in https://github.com/netglade/auto_mappr/pull/59/commits/badcdaca80e38d4135b2a71077bbbc094645fc63 and since the comment isn't relevant for consumers of the function it's intentionally a double slash one just for someone who might edit the code later.

Reply:
Seems all right.